soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1002,14 +1009,19 @@ void Randomizer::ParseItemLocationsFile(const char* spoilerFileName, bool silent
                     // todo handle prices
                     if (itemit.key() == "item") {
                         gSaveContext.itemLocations[index].check = SpoilerfileCheckNameToEnum[it.key()];
-                        gSaveContext.itemLocations[index].get = SpoilerfileGetNameToEnum[itemit.value()];
+                        gSaveContext.itemLocations[index].get.rgID = SpoilerfileGetNameToEnum[itemit.value()];
                     } else if (itemit.key() == "price") {
                         merchantPrices[gSaveContext.itemLocations[index].check] = itemit.value();
```

Comment:
I know it extends the scope of this, but do you think this same system could be used to store the prices or do you foresee problems?

Reply:
Something along those lines yeah, but I also kind of want to move most of the randomzier data out of gSaveContext later anyway and I can move prices then. Want them stored in a C++ class at runtime so we can reduce duplication (just storing the itemLocations map instead of that and the array in gSaveContext) and not have to store hardcoded char arrays or mess with `c_str()`s.